manifests/rule/remote.pp

```diff
@@ -309,6 +309,51 @@
       }
     }
 
+    # Basic validation for the action queue parameters
+    if $queue_size {
+      # First check the queue size, which should not be less than 100 per the docs
+      if $queue_size < 100 {
+        # Warn the user about an errant configuration, but don't fail as RSyslog will still run when setup this way
+        notify { "Invalid queue_size specified for ${name}":
```

Comment:
Is there a reason to use a `notify` resource instead of the `warn()` function?

Agents applying this code will never run green, because notify messages are considered resource changes.